app/serializers/npq_application_csv_serializer.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require "csv"
+
+class NPQApplicationCsvSerializer
+  attr_reader :scope
+
+  def self.headers
+    %w[
+      id
+      participant_id
+      full_name
+      email
+      email_validated
+      teacher_reference_number
+      teacher_reference_number_validated
+      school_urn
+      headteacher_status
+      eligible_for_funding
+      funding_choice
+      course_id
+      course_name
+    ]
+  end
+
+  def initialize(scope)
+    @scope = scope
+  end
+
+  def call
+    CSV.generate do |csv|
+      csv << self.class.headers
+
+      scope.each do |record|
+        csv << to_row(record)
+      end
+    end
+  end
+
+private
+
+  def to_row(record)
+    [
+      record.id,
+      record.user_id,
+      record.user.full_name,
+      record.user.email,
+      true,
```

Comment:
Are we sure email will always be validated? If not, can we have a todo comment here?

Reply:
Yes. I don't believe it is possible to complete the journey to submit an application until they have confirmed their email address on NPQ side